tests/phpunit/tests/xmlrpc/wp/newComment.php

```diff
@@ -8,37 +8,63 @@ class Tests_XMLRPC_wp_newComment extends WP_XMLRPC_UnitTestCase {
 	function test_valid_comment() {
 		$this->make_user_by_role( 'administrator' );
 		$post = self::factory()->post->create_and_get();
- 
-		$result = $this->myxmlrpcserver->wp_newComment( array( 1, 'administrator', 'administrator', $post->ID, array(
-			'content' => rand_str( 100 )
-		) ) );
- 
+
+		$result = $this->myxmlrpcserver->wp_newComment(
+			array(
+				1,
+				'administrator',
+				'administrator',
+				$post->ID,
+				array(
+					'content' => rand_str( 100 ),
+				),
+			)
+		);
+
 		$this->assertNotIXRError( $result );
 	}
- 
+
 	function test_empty_comment() {
 		$this->make_user_by_role( 'administrator' );
 		$post = self::factory()->post->create_and_get();
- 
-		$result = $this->myxmlrpcserver->wp_newComment( array( 1, 'administrator', 'administrator', $post->ID, array(
-			'content' => ''
-		) ) );
- 
+
+		$result = $this->myxmlrpcserver->wp_newComment(
+			array(
+				1,
+				'administrator',
+				'administrator',
+				$post->ID,
+				array(
+					'content' => '',
+				),
+			)
+		);
+
 		$this->assertIXRError( $result );
 		$this->assertEquals( 403, $result->code );
 	}
 
 	function test_new_comment_post_closed() {
 		$this->make_user_by_role( 'administrator' );
-		$post = self::factory()->post->create_and_get( array(
-			'comment_status' => 'closed'
-		) );
+		$post = self::factory()->post->create_and_get(
+			array(
+				'comment_status' => 'closed',
+			)
+		);
 
 		$this->assertEquals( 'closed', $post->comment_status );
 
-		$result = $this->myxmlrpcserver->wp_newComment( array( 1, 'administrator', 'administrator', $post->ID, array(
-			'content' => rand_str( 100 ),
-		) ) );
+		$result = $this->myxmlrpcserver->wp_newComment(
+			array(
+				1,
+				'administrator',
+				'administrator',
+				$post->ID,
+				array(
```

Comment:
Should be on one line since it's only one item.

Reply:
and the one above also since it's not an associative array